### PR TITLE
Fix to doTestCreateDeleteAddressesViaAgentLong that will enable scree…

### PR DESCRIPTION
**Pull request by RH-sdavey**
…nshots to be saved in ci

### CODE REVIEW

File: systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java [hunk 1]
```diff
@@ -310,9 +310,16 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
         ArrayList<Destination> addresses = generateQueueTopicList("via-web", IntStream.range(0, addressCount));
 
         runTestInLoop(30, () -> {
-            consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
-            consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
-            Thread.sleep(5000);
+            try {
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java]: I think we should add there this lines

```
selenium.saveScreenShots("testClass", "testName");
selenium.tearDownDrivers()
```

WDYT?

Comment (RH-sdavey) [reply systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java]: Yes, youre right man, as 'onfailed' will not be called in this situation anymore, good call. 
I have added them

File: systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java [hunk 2]
```diff
@@ -310,9 +310,16 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
...
+                consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
+                consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
+                Thread.sleep(5000);
+            } catch (Exception ex) {
+                selenium.setupDriver(environment, kubernetes, buildDriver());
+                consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(addressSpace), addressApiClient, addressSpace, user);
+                consoleWebPage.openWebConsolePage(user);
+                throw new Exception(ex);
```
Comment (RH-sdavey) [inline systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java]: and I suppose I left this throw exception here by mistake during my testing, I will remove it

File: systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java [hunk 3]
```diff
@@ -310,9 +310,17 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
...
+            try {
+                consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
+                consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
+                Thread.sleep(5000);
+                selenium.saveScreenShots("testClass", "testName");
+                selenium.tearDownDrivers();
+            } catch (Exception ex) {
+                selenium.setupDriver(environment, kubernetes, buildDriver());
+                consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(addressSpace), addressApiClient, addressSpace, user);
+                consoleWebPage.openWebConsolePage(user);
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java]: Why did you delete exception throwing?

File: systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java [hunk 4]
```diff
@@ -310,9 +310,17 @@ void doTestCreateDeleteAddressesViaAgentLong(AddressSpace addressSpace) throws E
...
 
         runTestInLoop(30, () -> {
-            consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
-            consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
-            Thread.sleep(5000);
+            try {
+                consoleWebPage.createAddressesWebConsole(addresses.toArray(new Destination[0]));
+                consoleWebPage.deleteAddressesWebConsole(addresses.toArray(new Destination[0]));
+                Thread.sleep(5000);
+                selenium.saveScreenShots("testClass", "testName");
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/marathon/MarathonTestBase.java]: Sorry for confusing, with "testClass" and "testName" I mean a real testclass and testname
